automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -175,7 +197,10 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
   // TODO This is a very weak check to deterimine if page is ready
   def isLoaded: Boolean = {
     await spinner "Loading attributes..."
-    openLaunchAnalysisModalButton.isVisible
+    // checks button is visible and has a value in its `data-test-state` attribute.
+    // Doesn't matter what value of `data-test-state` is, cares only if attribute value exists.
+    // Evaluate to false if `isVisible` evaluate to false.
+    openLaunchAnalysisModalButton.isVisible && !openLaunchAnalysisModalButton.getState.isEmpty
```

Comment:
nit: I prefer `openLaunchAnalysisModalButton.getState.nonEmpty` over `!openLaunchAnalysisModalButton.getState.isEmpty` for readability, even though they are functionally equivalent.

Reply:
I also still think this could be moved up into the `Button` class, but I'm not gonna hold the PR  for that.

Reply:
`isEmpty` replaced with `nonEmpty`. I'm hesitating to create a new method in `Button` class because I don't like to clutter class with a "used one-time" method.